resources/options_lookup.tsv

```diff
@@ -1150,9 +1150,12 @@ Water Heater	"Electric Heat Pump, 80 gal"	ResidentialHotWaterHeaterHeatPump	stor
 		ResidentialAirflow	has_water_heater_flue=false																																															
 Water Heater	FIXME Other Fuel	ResidentialHotWaterHeaterTank	fuel_type=gas	tank_volume=auto	setpoint_temp=125	location=auto	capacity=40	energy_factor=0.59	recovery_efficiency=0.76	offcyc_power=0	oncyc_power=0																																							
 		ResidentialAirflow	has_water_heater_flue=false																																															
-Hot Water Fixtures	50%	ResidentialHotWaterFixtures	shower_mult=0.5	sink_mult=0.5	bath_mult=0.5	plant_loop=auto																																												
-Hot Water Fixtures	100%	ResidentialHotWaterFixtures	shower_mult=1.0	sink_mult=1.0	bath_mult=1.0	plant_loop=auto																																												
-Hot Water Fixtures	200%	ResidentialHotWaterFixtures	shower_mult=2.0	sink_mult=2.0	bath_mult=2.0	plant_loop=auto																																												
+Hot Water Fixtures	50% Usage	ResidentialHotWaterFixtures	shower_mult=0.5	sink_mult=0.5	bath_mult=0.5	plant_loop=auto																																												
+Hot Water Fixtures	"50% Usage, Low Flow"	ResidentialHotWaterFixtures	shower_mult=0.48	sink_mult=0.48	bath_mult=0.5	plant_loop=auto																																												

```

Comment:
I don't know if it will matter, but why are there quotes only around the low flow options? If you could just remove those I think everything looks fine

Reply:
When you save in Excel, it slaps quotes around any options with a comma in it. Annoying, but doesn't matter.